src/jit/gentree.cpp

```diff
@@ -1915,8 +1926,18 @@ unsigned Compiler::gtHashValue(GenTree* tree)
                 break;
         }
 
-        // narrowing cast, but for hashing.
-        hash = genTreeHashAdd(hash, (unsigned)add);
+        // clang-format off
```

Comment:
Why do you need to turn off clang-format? seems like pretty simple formatting.

Reply:
Because it gets it wrong here:  It wants me to add 9 spaces

Reply:
// clang-format off
        // narrow 'add' into a 32-bit 'val'
        unsigned val;
#ifdef _TARGET_64BIT_
        val = genTreeHashAdd(uhi32(add), ulo32(add));
#else // 32-bit target
        val = add;
#endif
        // clang-format on